docs/src/assets/configuration.yaml

```diff
@@ -0,0 +1,11 @@
+cluster-config:
+     network:
+       enabled: true
+     dns:
+       enabled: true
+     local-storage:
+       enabled: true
+   extra-node-kubelet-args:
+     --reserved-cpus: "0-31"
+     --cpu-manager-policy: "static"
+     --topology-manager-policy: "best-effort"
```

Comment:
nit: please add a newline - otherwise GH complains with this red circle

Reply:
Fixed thanks for pointing it out